test/e2e/pod.go

```diff
@@ -39,23 +39,19 @@ var _ = ginkgo.Describe("Pod to external server PMTUD", func() {
 	// for both HTTP and UDP and different ingress and egress payload sizes.
```

Comment:
haha I (apparently I wrote this test but cannot remember anything :/) clearly tell:
`// This traffic scenario mimics a pod2external setup where large packets and needs frag is involved.`
so I think you are doing the right thing here by changing this to actually use an external container/server.